js/task-tracker.js

```diff
@@ -65,8 +76,8 @@ class Task {
         `;
 
         html.style.cssText = `
-            color: ${themes[this.theme].color};
-            background-color: ${themes[this.theme].background};
+            color: ${themes[priorities[this.priority]].color};
```

Comment:
I'd create `const theme getTheme(this.priority)`
and use here like `theme.color` and etc.

Reply:
thanks. it should be correct. I created methods `getPriority()`, `getTextColor()` and `getBackgroundColor()` in class `Task`